tests/integration/test_connection.py

```diff
@@ -0,0 +1,42 @@
+import threading
+import unittest
+from unittest.mock import patch
+
+from kuyruk import Kuyruk
+from tests import tasks
+from tests.integration.util import delete_queue, drop_connections, run_worker
+
+
+class ConnectionTestCase(unittest.TestCase):
```

Comment:
I also wanted to add a case for task calling another task, but couldn't make it work (neither with the old behavior).